tensorboard/webapp/runs/views/runs_selector/runs_selector_component.ts

```diff
@@ -25,6 +25,13 @@ import {RunsTableColumn} from '../runs_table/types';
   `,
   styles: [
     `
+      :host {
+        display: block;
+        height: 100%;
```

Comment:
A minor preference: If we leave height unset then for fewer runs the horizontal scrollbar will be closer to the runs instead at the bottom of the page.

Up to you.

Reply:
I tried this out. Unfortunately this also prevents vertical scroll entirely on longer tables - the height 100% is required to *constrain* the long table height to enable scrolling